services/src/main/java/org/apache/druid/cli/PullDependencies.java

```diff
@@ -141,12 +142,24 @@
       "org.roaringbitmap",
       "net.java.dev.jets3t"
       */
-  );
 
   private static final List<String> DEFAULT_REMOTE_REPOSITORIES = ImmutableList.of(
       "https://repo1.maven.org/maven2/"
   );
 
+  // These are either not needed or are provided by core
+  private static final Exclusions HADOOP_DEFAULT_SECURITY_VULNERABILITY_EXCLUSIONS =
```

Comment:
Can we split this to have one set of security exclusions and another set for provided-by-core exclusions?

Reply:
I've split it into two groups